app/main.py

```diff
@@ -1,6 +1,21 @@
+import os
+import requests
+from dotenv import load_dotenv
+
+load_dotenv()
+
+KEY = os.getenv("API_KEY")
+URL = "http://api.weatherapi.com/v1/current.json"
+LOCATION = "Paris"
+
+
 def get_weather() -> None:
-    # write your code here
-    pass
+    response = requests.get(URL, params={"q": LOCATION, "key": KEY}).json()
```

Comment:
The API key and location are directly embedded into the URL. It's a good practice to separate the base URL from the parameters. This way, if you need to change the parameters or add new ones, you don't need to modify the base URL. Consider using the `params` argument of the `requests.get` function to pass parameters to the API.